laravel-app/resources/views/layouts/app.blade.php

```diff
@@ -14,23 +14,27 @@
     <link href="https://fonts.bunny.net/css?family=Nunito" rel="stylesheet">
```

Comment:
why <head></head>are not separated in the other file? create a new file and use @include instead.

Reply:
Separated into files sir, used @include instead. Thank you ❤️